.github/workflows/tests.yml

```diff
@@ -48,6 +48,9 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Restore dependencies
+        uses: ./.github/workflows/actions/install-node
```

Comment:
Had to move this line here for Windows to find `node_modules/.bin/tsc`

Unlike our Linux runners its not in the global “npm get cache” location at `~/.npm`

```
> build:types
> tsc --build tsconfig.build.json

'tsc' is not recognized as an internal or external command,
operable program or batch file.
```